src/components/quoteWithImage/quoteWithImage.module.css

```diff
@@ -1,9 +1,12 @@
 .background {
-  background-blend-mode: multiply;
   background-color: #b00e0c;
   width: 100%;
 }
 
+.background::after {
```

Comment:
Thanks for the feedback @jlhogan! I tried applying styling directly to the `.background.’ selector but the red color just wouldn’t apply. I’m assuming because it’s an image. By adding the `::after` selector I could add the red color “after” the background and it works the same apparently! CSS is soooo strange haha